.changeset/grumpy-planets-sniff.md

```diff
@@ -0,0 +1,5 @@
+---
+"@neo4j/graphql": patch
+---
+
+Integration tests to cover bugs highlighted by #2068
```

Comment:
:thinking: I reckon this doesn't need a changelog, as this is just internal tests, no fixes or features. Thoughts?

Reply:
I don't really have a strong opinion on this. I'm happy to remove it if that's what you think is best